smarts/core/sensors/__init__.py

```diff
@@ -708,14 +708,20 @@ def _vehicle_is_off_route_and_wrong_way(
         )
         # Check that center of vehicle is still close to route
         radius = vehicle_minimum_radius_bounds + 5
-        nearest_lane = sim_local_constants.road_map.nearest_lane(
+        nearest_lanes = sim_local_constants.road_map.nearest_lanes(
             vehicle_pos, radius=radius
         )
 
         # No road nearby, so we're not on route!
-        if not nearest_lane:
+        if not nearest_lanes:
             return (True, False)
 
+        # Handle case where there are multiple nearest lanes the same dist away
+        min_dist = nearest_lanes[0][1]
+        tied_nearest = [lane for (lane, d) in nearest_lanes if d == min_dist]
```

Comment:
I could suggest an allowance for slight deviation.
```suggestion
        tied_nearest = [lane for (lane, d) in nearest_lanes if math.isclose(d, min_dist)]
```